frontend/src/components/Step4.tsx

```diff
@@ -71,44 +78,39 @@ export const Step4: React.FC<StepProps> = ({ next }: StepProps) => {
                   type="text"
                   name="licenseNumber"
                   id="licenseNumber"
-                  value={licenseNumber}
-                  onChange={(e) => {
-                    setLicenseNumber(e.target.value);
-                  }}
+                  value={formData.licenseNumber}
+                  onChange={handleInputChange}
                   placeholder="Enter Certified/Registered/License Number"
                 />
               </label>
             </div>
 
             <div className={styles.inputBox}>
-              <label htmlFor="state" className={styles.inputTitle}>
+              <label htmlFor="jurisdictionName" className={styles.inputTitle}>
                 Name of State/Jurisdiction
                 <input
                   className={styles.inputText}
                   type="text"
-                  name="state"
-                  id="state"
-                  value={state}
-                  onChange={(e) => {
-                    setState(e.target.value);
-                  }}
+                  name="jurisdictionName"
+                  id="jurisdictionName"
+                  value={formData.jurisdictionName}
+                  onChange={handleInputChange}
                   placeholder="Enter Name of State/Jurisdiction"
                 />
               </label>
             </div>
 
             <div className={styles.inputBox}>
-              <label htmlFor="licenseExpiration" className={styles.inputTitle}>
+              <label htmlFor="dateCertified" className={styles.inputTitle}>
```

Comment:
this date Certified is actually still an expiration date.
When creating the context. name the value to `dateCertifiedExpires` same in the context